prow/gerrit/client/client.go

```diff
@@ -236,6 +239,11 @@ func (c *Client) GetBranchRevision(instance, project, branch string) (string, er
 	return res.Revision, nil
 }
 
+// Account returns gerrit account for the given instance
+func (c *Client) Account(instance string) *gerrit.AccountInfo {
+	return c.accounts[instance]
```

Comment:
check instance exists?

Reply:
we want to return nil in that case anyway